src/main/java/org/dependencytrack/persistence/DefaultObjectGenerator.java

```diff
@@ -244,13 +265,23 @@ private void loadDefaultConfigProperties() {
     private void loadDefaultNotificationPublishers() {
         try (QueryManager qm = new QueryManager()) {
             LOGGER.info("Synchronizing notification publishers to datastore");
-            for (final DefaultNotificationPublishers publisher : DefaultNotificationPublishers.values()) {
-                try {
-                    NotificationUtil.loadDefaultNotificationPublishers(qm);
-                } catch (IOException e) {
-                    LOGGER.error("An error occurred while synchronizing a default notification publisher", e);
-                }
-            }
+            NotificationUtil.loadDefaultNotificationPublishers(qm);
+        } catch (IOException e) {
+            LOGGER.error("An error occurred while synchronizing a default notification publisher", e);
         }
     }
+
+    String generateEnvVariableName(ConfigPropertyConstants configProperty) {
+        StringBuilder sb = new StringBuilder();
+        sb.append(configProperty.getGroupName().toUpperCase().replaceAll("[\\-\\.]", "_"));
```

Comment:
Should we require a common prefix here, too? Similar to `DEPENDENCY_TRACK_ADMIN_USERNAME`.

Perhaps `DT` would be a better choice than `DEPENDENCY_TRACK`, to keep the length reasonable :D

Reply:
I thought about it and didn't add a prefix out of pure lazyness. For consistency check it would require env variable like  `DEFAULT_TEMPLATES_OVERRIDE_ENABLED` to have the prefix also.
I'll add the `DT` prefix to match alpine behaviour.